assets/panel/debugger.properties

```diff
@@ -240,6 +240,10 @@ breakpointMenuItem.disableAll=Disable all breakpoints
 breakpointMenuItem.disableAll.accesskey=k
 breakpointMenuItem.deleteAll=Remove all breakpoints
 breakpointMenuItem.deleteAll.accesskey=a
+breakpointMenuItem.removeCondition=Remove breakpoint condition
```

Comment:
breakpointMenuItem.removeCondition.label (I know devtools hasn't used this naming before, but it's clearer than just adding .accesskey to the access key).